Framework/DataHandling/inc/MantidDataHandling/LoadShape.h

```diff
@@ -0,0 +1,83 @@
+#ifndef DATAHANDLING_LOAD_SHAPE_H_
+#define DATAHANDLING_LOAD_SHAPE_H_
+
+#include "MantidAPI/IFileLoader.h"
+#include "MantidGeometry/Objects/MeshObject.h"
+#include "MantidKernel/V3D.h"
+
+namespace Mantid {
+namespace DataHandling {
+/**  Load Shape into an instrument of a workspace
```

Comment:
Load a shape for the sample on the an experiment info of the workspace. More detail that that should be provided.

Reply:
What details do you want adding? Which questions would you want these details to answer?